### PR TITLE
Update k8s dependencies to v1.19.0

### PR DESCRIPTION
**Pull request by hakman**
/cc @rifelpet

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -2,49 +2,49 @@ module k8s.io/kops
...
-replace k8s.io/cli-runtime => k8s.io/cli-runtime v0.19.0-rc.4
+replace k8s.io/cli-runtime => k8s.io/cli-runtime v0.19.0
 
-replace k8s.io/kube-controller-manager => k8s.io/kube-controller-manager v0.19.0-rc.4
+replace k8s.io/kube-controller-manager => k8s.io/kube-controller-manager v0.19.0
 
-replace k8s.io/code-generator => k8s.io/code-generator v0.19.0-rc.4
+replace k8s.io/code-generator => k8s.io/code-generator v0.19.0
 
 replace github.com/gophercloud/gophercloud => github.com/gophercloud/gophercloud v0.11.0
```
Comment (olemarkus) [inline go.mod:49]: Could you try remove this line? It shouldn't be necessary now that k8s doesn't directly depend on it. gomod should hopefully just use the version we specify and there should only be one line in go.sum.

Comment (hakman) [reply go.mod:49]: Not yet :)
```
go: k8s.io/cloud-provider-openstack@v1.18.0 requires
	k8s.io/kubernetes@v1.18.0 requires
	k8s.io/code-generator@v0.0.0: reading k8s.io/code-generator/go.mod at revision v0.0.0: unknown revision v0.0.0
```

Comment (hakman) [reply go.mod:49]: https://github.com/kubernetes/cloud-provider-openstack/pull/1151

**Pull request merged**